packages/design-system/snackbar/snackbar-container.tsx

```diff
@@ -0,0 +1,27 @@
+import { AnimatePresence } from "moti";
+import { FullWindowOverlay } from "react-native-screens";
+
+import { Snackbar, SnackbarProps, SNACKBAR_HEIGHT } from "./snackbar";
+
+export const SnackbarContainer = ({
+  show,
+  snackbar,
+  ...rest
+}: SnackbarProps) => {
+  return (
+    <AnimatePresence>
+      {show && (
+        <FullWindowOverlay
```

Comment:
FullWindowOverlay only works on iOS. On android, it's a normal [View](https://github.com/software-mansion/react-native-screens/blob/3b4d2641b1b757e25a6dabccf78d238d5f31f661/src/index.native.tsx#L465). 🙁

Reply:
We can use `Modal` on android but I think it'll block background View's touches. I am not very sure though, will need to test

Reply:
oh my bad, I don't know `FullWindowOverlay ` only works on iOS, so I will try `Modal` on android.
but at the moment on Android, I think using `View` may be enough, because `ModalScreen` is just a normal JS screen, right? 
anyway, will try once.

Reply:
> I think using View may be enough

yeah, I forgot that it's mounting in the provider which is at the top. Still, it won't show up on top of Modal for e.g. But I think it's fine for now. We don't have that usecase yet.